src/js/gui.js

```diff
@@ -7,8 +7,6 @@ import $ from "jquery";
 const TABS = {};
 
 const GUI_MODES = {
-    NWJS: "NW.js",
-    Cordova: "Cordova",
     Other: "Other",
```

Comment:
The GUI_MODES is being used at other places? It has not too much sense having only one option

Reply:
@McGiverGim thanks updated.